concepticondata/conceptlists.tsv

```diff
@@ -1,6 +1,7 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
 Zhao-2006-201	Zhào, Yànzhēn 趙燕珍	2006		201	basic	Chinese	Zhaozhuang Bai		Zhao2006		This is a concept list prepared to illustrate basic lexemes in Zhaozhuang Bai, a Southern Bai variety spoken in the South of Dali in Yunnan (China). The author intended to use Swadesh's [list of 200 items](:ref:Swadesh-1952-200), but there are a few derivations. Interestingly, some concepts are also identical (like the Chinese items for SKY and DAY (NOT NIGHT)), due to ambiguities in the glossing language. Furthermore, as in many Chinese lists, we observe a wrong translation from DULL, being rendered as STUPID rather than DULL of a knife.		
 Savelyev-2020-254	Savelyev, Alexander and Robbeets, Martine	2020		254	basic	English	Turkic languages	https://zenodo.org/record/3555174/files/turkic_alignment.tsv?download=1	Savelyev2020		This concept list, which extends the classical [Swadesh list of 200 items](:ref:Swadesh-1952-200) by the [Leipzig-Jakarta list](:ref:Tadmor-2009-100) was used for a phylogenetic study on Turkic languages.		
+Oskolskaya-2021-254	Oskolskaya, Sofia and Koile, Ezequiel and Robbeets, Martine	2021		254	basic	English	Tungusic languages	https://benjamins.com/catalog/dia.20010.osk/additional	Oskolskaya2021		This concept list, which extends the classical [Swadesh list of 200 items](:ref:Swadesh-1952-200) by the [Leipzig-Jakarta list](:ref:Tadmor-2009-100) was used for a phylogenetic study on Tungusic languages. It is identical to [Saveleyev and Robbeets (2020)](:ref:Savelyev-2020-254) except for concept numbering.		
```

Comment:
Can I ask you to write "is supposed to extend", as we already found out that this is not the case, since they use terms that occur neither in Leipzig-Jakarta nor in Swadesh 200 ?

Reply:
done (and updated S2020 to match)